GRDB/Core/Database.swift

```diff
@@ -317,11 +269,30 @@ extension Database {
 extension Database {
 
     // MARK: - Database Setup
-    
+    func setupConnection() throws {
+        do {
+            try Database.activateExtendedCodes(sqliteConnection)
+            #if SQLITE_HAS_CODEC
+            try Database.validateSQLCipher(sqliteConnection)
+            #endif
+            if let passphrase = configuration.passphrase {
+                try Database.set(passphrase: passphrase, forConnection: sqliteConnection)
```

Comment:
This call should be guarded by `#if SQLITE_HAS_CODEC`.

Reply:
Good catch, thanks!